proxy/envoy/watcher.go

```diff
@@ -117,13 +117,20 @@ func NewProxy(config proxyconfig.ProxyConfig, node string) proxy.Proxy {
 }
 
 func (proxy envoy) args(fname string, epoch int) []string {
-	return []string{"-c", fname,
+	startupArgs := make([]string, 0)
```

Comment:
make and first append not needed

Reply:
yeah, just store as startupArgs := []string{...} and then append.